src/util/custom.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module "*.svg" {
```

Comment:
this is handy – I didn't look for how to do this, and instead used `require` – I can correct that once both PRs are merged